bindata/v3.11.0/openshift-apiserver/ds.yaml

```diff
@@ -101,6 +134,12 @@ spec:
       - name: serving-cert
         secret:
           secretName: serving-cert
+      - name: trusted-ca-bundle
```

Comment:
should this be optional?

Reply:
Yes, given a message from proxy people yesterday, non-optional CMs like this may prolong upgrades since the network operator may start injecting the CMs too late

Reply:
Right answer was no :-) We want to require this.

Reply:
Apparently yes was still the good answer :)) There's people who want us to have it optional for the reasons above, this is now handled by bash guard in the pod's `commands`

Reply:
optional is needed to support 4.1>4.2 upgrades.